tests/baselines/reference/declarationEmitInvalidExport.errors.txt

```diff
@@ -1,14 +1,11 @@
-declarationEmitInvalidExport.ts(4,30): error TS4081: Exported type alias 'MyClass' has or is using private name 'myClass'.
 declarationEmitInvalidExport.ts(5,1): error TS1128: Declaration or statement expected.
 
 
-==== declarationEmitInvalidExport.ts (2 errors) ====
+==== declarationEmitInvalidExport.ts (1 errors) ====
     if (false) {
       export var myClass = 0;
     }
     export type MyClass = typeof myClass;
-                                 ~~~~~~~
-!!! error TS4081: Exported type alias 'MyClass' has or is using private name 'myClass'.

```

Comment:
Declaration emit errors are emitted in the presence of parse errors (unlike checker errors), so this (pretty bogus, given the parse error) error was here before, and now it's gone. If checker errors were reported, there would be one on `myClass` here, but, again, blocked by a parse error anyway.